File: 268-missing-number/missing.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        for num in range(len(nums)+1):
            if num not in nums:
                return num  

        # return sum(range(len(nums)+1)) - sum(nums)

        # n = len(l)
        # return ((n * (n+1)) // 2) - sum(l) 

File: 268-missing-number/missing.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var missingNumber = function(nums) {
    let full =[]
    for (let i=0;i < nums.length+2;i++){
            full.push(i)
    }
    for (let num of full){
        if (!nums.includes(num)){return num}
    }
};